src/webgpu/api/validation/render_pass/attachment_compatibility.spec.ts

```diff
@@ -184,14 +184,7 @@ g.test('render_pass_and_bundle,color_format')
     const { passFormat, bundleFormat } = t.params;
 
     t.skipIfTextureFormatNotSupported(passFormat, bundleFormat);
-    // float16 and float32 might miss color renderablity under compat mode
-    t.skipIf(
-      t.isCompatibility &&
-        (is16Float(passFormat) ||
-          is32Float(passFormat) ||
-          is16Float(bundleFormat) ||
-          is32Float(bundleFormat))
-    );
+    t.skipIfMultisampleNotSupportedForFormat(passFormat, bundleFormat);
```

Comment:
I thought this was checking for renderable, not multisample?

Reply:
Oops. Thanks for catching that. Fixed now.